src/tuple_gen.h

```diff
@@ -886,6 +928,16 @@ struct CTupleGenYamlInfo {
         }
         return 0;
     }
+    
+    void set_server_track_ports(uint8_t svr_pool_id, uint8_t plugin_id) {
+        if (plugin_id!=0) {
+            if (m_server_pool[svr_pool_id].m_is_bundling==false) {
```

Comment:
Not all plugin need to track ports.
Maybe better to have an explicit YAML section.
And you should verify that in that case user choose server with tracking ports

Hanoh
Sent from my iPhone

On 17 ????? 2015, at 8:24, Wenxian Li <notifications@github.com<mailto:notifications@github.com>> wrote:


In src/tuple_gen.h<https://github.com/cisco-system-traffic-generator/trex-core/pull/4#discussion_r45026017>:

> @@ -886,6 +928,16 @@ struct CTupleGenYamlInfo {
>          }
>          return 0;
>      }
> +
> +    void set_server_track_ports(uint8_t svr_pool_id, uint8_t plugin_id) {
> +        if (plugin_id!=0) {
> +            if (m_server_pool[svr_pool_id].m_is_bundling==false) {


If user configures plugin_id and not configures track_ports, I'll force to enable track_ports.
I'm thinking do we need such logic? Not every plugin needs to track server port, right?

Or I can simply print out a warning without force to enable track_ports?

-
Reply to this email directly or view it on GitHub<https://github.com/cisco-system-traffic-generator/trex-core/pull/4/files#r45026017>.